docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -87,19 +90,26 @@ The following example shows how to create a gallery-like flexbox layout using th
     content.JustifyContent = FlexContainer.Justification.JustifyCenter;
     /// Align items vertically center
     content.AlignItems = FlexContainer.Alignment.AlignCenter;
-    /// Occupy 90 percent of available space on the cross axis
-    content.Flex = 0.9f;
     /// Add the content area to the main container
     flexContainer.Add(content);
     ```
 
-7.  Add items to the toolbar.
+7.  Use `Flex` property of the `FlexContainer` class to specify the area to be occupied within the main container. Since the main container displays items vertically its height is shared between the toolbar and the content. Setting `toolbar.Flex` to 0.1 and `content.Flex` to 0.9 means that `toolbar` will occupy 10% of vertical space and `content` will occupy 90% of vertical space of the main container. Width of items will be eqaul to width of main container.
+
+    ```csharp
+    /// Occupy 10 percent of available space on the cross axis
+    toolBar.Flex = 0.1f;
+    /// Occupy 90 percent of available space on the cross axis
+    content.Flex = 0.9f;
+    ``` 
+
+8.  Add items to the toolbar.
 
     The toolbar has 1 button on the left, 1 button on the right, and a title in the center. To keep the title in the center, make the title flexible so that it automatically takes all remaining horizontal space. To make the layout look nicer, also add some space around the items.
 
     Creating the toolbar with a flex container allows you to keep the toolbar layout consistent, even as it expands or reduces in size based on screen size and orientation.
```

Comment:
Global: 
If the sentences are/is immediately followed by code or image, request you to ensure to add : instead of . at the end of the sentences.
```suggestion to 
    Creating the toolbar with a flex container allows you to keep the toolbar layout consistent, even as it expands or reduces in size based on screen size and orientation: